streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -159,6 +159,7 @@ public void testGetGroupInstanceIdConfigs() {
         assertNull(returnedProps.get(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG));
     }
 
+    @SuppressWarnings("deprecation") // TODO revisit in follow up PR
```

Comment:
Hey, sorry if this is picky, but are these also to be removed when we remove the config? If so, maybe we should just say it on every suppression.

Reply:
Not 100% sure yet how the test will be rewritten. My expectation is, that we remove the `retries` config and also remove the `Suppress` tag -- however, we can only do this, after streams code was changes.

Reply:
Addressed via #9060